packages/appstore/src/components/appstore-banner/appstore-banner.tsx

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 import SVGMigrationBanner from './svg-migration-banner';
+import { useMT5SVGEligibleToMigrate } from '@deriv/hooks';
 
 const AppstoreBanner = () => {
+    const { has_svg_accounts_to_migrate } = useMT5SVGEligibleToMigrate();
     //TODO: will add proper condition check when API is ready
-    if (true) return <SVGMigrationBanner />;
+    if (has_svg_accounts_to_migrate) return <SVGMigrationBanner />;
+
+    return null;
```

Comment:
can we simplify this:

  return has_svg_accounts_to_migrate && <SVGMigrationBanner />;

Reply:
This will cause the imports in `main-title-bar.index.tsx` to have Error when it state like this. will add as follow: `return (has_svg_accounts_to_migrate && <SVGMigrationBanner />) || null;` is it ok?

Reply:
`return has_svg_accounts_to_migrate ? <SVGMigrationBanner /> : null;`
Then isnt this better?

Reply:
will change it 😎